### DISCUSSION TITLE
How can I update the docker-compose version?

### DISCUSSION THREAD

**Discussion by drik98** [podman-desktop/podman-desktop#6553] [Q&A] [answered]
I'd like to use feature available on from docker-compose version >=2.24.0. The docker compose version displayed in podman desktop is v2.23.3 while the latest current docker-compose version is [v2.26.0](https://github.com/docker/compose/releases/tag/v2.26.0). 

![image](https://github.com/containers/podman-desktop/assets/36853386/68552679-6304-401f-b294-09cc57820b38)

Is it possible to update the docker compose version or do I have to wait until it is available through podman?

**Comment by benoitf** [comment, marked as answer]
it looks like there is a bug / missing thing

you can delete `~/.local/share/containers/podman-desktop/extensions-storage/podman-desktop.compose/bin/docker-compose` and `/usr/local/bin/docker-compose` and then restart Podman Desktop, in Settings/Resources page, click on 'Setup...' page it'll download a new version

**Reply by drik98** [reply]
That was it. Thank you very much!
